wchat/screens/GroupList/components/GroupCell/index.js

```diff
@@ -1,16 +1,16 @@
 import React, { Component } from 'react';
-import { AppRegistry, Text, Image, View } from 'react-native';
+import { AppRegistry, Text, TouchableOpacity, Image, View } from 'react-native';
 import styles from './styles.js'
 
 export default class GroupCell extends Component {
```

Comment:
if you dont have to handle functions or states just use function instead off class.